packages/replayio/src/utils/installation/installLatestRelease.ts

```diff
@@ -65,16 +74,10 @@ async function downloadReplayFile() {
   };
 
   for (let i = 0; i < 5; i++) {
-    console.log(
-      `Downloading ${highlight(runtimeMetadata.runtime)} from ${link("replay.io")}${
-        i ? ` ${dim(`(retry ${i} of 4)`)}` : ""
-      }`
-    );
-
```

Comment:
We lose a little bit of info here but I think in the overwhelming common case, the `logPromise` approach is a better UX.